x-pack/plugins/enterprise_search/public/applications/shared/enterprise_search_provider/index.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { createContext, FC, useContext } from 'react';
+
+import { InitialAppData } from '../../../../common/types';
+
+import { NO_ACCESS } from './constants';
+
+export interface EnterpriseSearchContextData {
+  initialData: InitialAppData;
+}
+const initialContextWithNoAccess: EnterpriseSearchContextData = {
+  initialData: {
```

Comment:
question: Why wrap this in an `initialData` object rather than just using a root-level access property? 

Also wondering whether this is really `initialData` or just `data` 🤔